recipes/ndcube/meta.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: ndcube
+  version: "0.1.dev273"
+
+source:
+  fn: ndcube-0.1.dev273.tar.gz
+  url: https://pypi.python.org/packages/21/e7/32f31309c75a064277babcdca88d6bb5dadefaea000303c61e177beba4d2/ndcube-0.1.dev273.tar.gz
```

Comment:
is there not a nicer URL you can get for the tarballs?

Reply:
Probably? That's the one `conda skeleton` gave me. Still not super-familiar with the inner workings of pypi, but I'll have a look.

Reply:
I think you can use the one from the example recipe:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18